inventory/src/__tests__/components/LoanReturn/ItemLoanReturn.test.js

```diff
@@ -0,0 +1,100 @@
+import '@testing-library/jest-dom/extend-expect';
+import { act, fireEvent, screen, waitFor } from '@testing-library/react';
+import { rest } from 'msw';
+import React from 'react';
+
+import { ItemLoanReturn } from '../../../components';
+import { Api } from '../../../globals';
+import { server } from '../../../mocks';
+import { activeItemLoans, activeKitLoans } from '../../../mocks/loanActive';
+import { render } from '../../../testSetup';
+import { getUrlWithoutParams } from '../../../utils';
+
+describe('Kit Loan Return', () => {
+  beforeEach(async () => {
+    server.use(
+      rest.get(getUrlWithoutParams(Api['kitHistory']), (req, res, ctx) => {
+        return res(ctx.status(200), ctx.json(activeKitLoans));
+      }),
+      rest.get(getUrlWithoutParams(Api['loan_active']), (req, res, ctx) => {
+        return res(ctx.status(200), ctx.json(activeItemLoans));
+      }),
+    );
+    act(() => {
+      render(<ItemLoanReturn />);
+    });
+    await waitFor(() => {
+      expect(screen.getAllByText('test')[0]).toBeInTheDocument();
+    });
+  });
+
+  afterEach(() => {
+    server.resetHandlers();
+  });
+
+  it('should have correct headings', async () => {
+    const headings = ['ID', 'Return Date', 'Loan Date', 'Loanee Name'];
+
+    for (const heading of headings) {
+      expect(screen.getAllByText(heading)[0]).toBeInTheDocument();
+    }
+  });
+
+  it('should display loan information correctly', async () => {
+    const details = ['125', '17/11/2023', '15/11/2023', 'test'];
+
+    for (const detail of details) {
+      expect(screen.getAllByText(detail)[0]).toBeInTheDocument();
+    }
```

Comment:
not too sure why it fails, but this seems to work

```suggestion
    screen.getByRole('button', { name: '125 11/17/2023 11/15/2023 test' });
```